### ISSUE TITLE
[Error] distrobox assemble replace hangs on deleting old container

### ISSUE THREAD

**Issue by jooooscha** [89luca89/distrobox#1052]
There are no open and several closed issues about this.
For example: #856 and #855, and loosely related also #841, #998.
They are closed with a reference to e18e01844e71d38af9b5cd67ac0e49d81cd735d2.
However, that commit does not fix the problem with `distrobox assemble`

**Describe the bug**

When running `asseble` with the `replace` option, the old container will be removed with `distrobox rm -f <container-name>` in: https://github.com/89luca89/distrobox/blob/bf65e627b7b5c9b505932a671618c378aa84c48f/distrobox-assemble#L223

The `-f` flag prevents the command from asking for confirmation of removal.
However, it still asks for confirmation to remove the home directory.
This should not happen, because the hom-dir-removal-confirmation is beind the `non_interactive` flag. Therefore, I strongly suspect that this flag is not correctly set on `distrobox asseble create`

**To Reproduce**

- Create `distrobox.ini`:
```
[archtest]
additional_packages="git neovim"
image=archlinux:latest
init=false
pull=true
root=false
replace=true
start_now=false
home=~/distrobox-home
```
- run `distrobox assemble create`
- Wait for successful creattion of container
- run `distrobox assemble create`
- Distrobox will print: `- Deleting archtest...` but the request for home-dir-removal will be piped to /dev/null.
- Distrobox hangs with no additional output

**Expected behavior**

- Container replaced

**Logs**
Run the commands with `--verbose` and post the log here as a file upload
Attach also the output of `podman logs` or `docker logs`, possibly with `--latest` flag

**Desktop (please complete the following information):**
 - Are you using podman, docker or lilipod? podman
 - Which version or podman, docker or lilipod? 4.5.0
 - Which version of distrobox? 1.5.0.2
 - Which host distribution? NixOS 23.05
 - How did you install distrobox? NixOS 23.05

**Additional context**

As far as i can tell, the `--yes` flag is need next to the `--force` flag in
https://github.com/89luca89/distrobox/blob/bf65e627b7b5c9b505932a671618c378aa84c48f/distrobox-assemble#L223

**Comment by 89luca89** [comment]
This is fixed already in git version
Just tried and works now

**Comment by jooooscha** [comment]
Just for the record: The does bug appear to be fixed on distrobox 1.6.0.1 (did not test 1.6.0)
